internal/fullnode/pvc_builder.go

```diff
@@ -79,9 +85,23 @@ func pvcResources(crd *cosmosv1.CosmosFullNode) corev1.ResourceRequirements {
 		reqs = crd.Spec.VolumeClaimTemplate.Resources
 		size = reqs.Requests[corev1.ResourceStorage]
 	)
+
 	if autoScale := crd.Status.SelfHealing.PVCAutoScale; autoScale != nil {
-		if autoScale.RequestedSize.Cmp(size) > 0 {
-			reqs.Requests[corev1.ResourceStorage] = autoScale.RequestedSize
+		// This is the implementation using Int64 but it does not support fractions
+		//requestSize, unableToConvert := autoScale.RequestedSize.AsInt64()
+		//if unableToConvert == true {
+		//	fmt.Errorf("Unable to convert auto scale request")
+		//} else {
+		//	sizeWithPadding := resource.NewQuantity(int64(float64(requestSize)*SnapshotScalingFactor), resource.DecimalSI)
+		//	if sizeWithPadding.Cmp(size) > 0 {
+		//		reqs.Requests[corev1.ResourceStorage] = *sizeWithPadding
+		//	}
+		//}
```

Comment:
So using int64 seems fine for volumes because they generally do not use fractions but them there is the failure branch that needs to be tested etc.

Reply:
Do we need the commented out code here or can it be deleted?

Reply:
I kept it for our discussion to show that ugly conditional - will delete